### ISSUE TITLE
About using the compute_meanDice for multi-organ segmentation

### ISSUE THREAD

**Issue by leeh43** [Project-MONAI/MONAI#2535]
Hi, I have a question on using the compute_meandice in the validation phase. I have followed the instruction of performing 3D spleen segmentation in tutorial and perform multi-organ segmentation on my training data. The training loss is decreasing, while it outputs Nan for computing the compute_meandice in the validation. Here is the screenshot of it and what do you recommend for me to change in the code?
![Screenshot from 2021-07-06 16-12-16](https://user-images.githubusercontent.com/54121206/124667408-fb61df00-de74-11eb-813c-1739bfda5ab8.png)
![Screenshot from 2021-07-06 16-13-21](https://user-images.githubusercontent.com/54121206/124667516-1f252500-de75-11eb-8800-7dc23a6b933f.png)

**Comment by wyli** [comment]
Hi, is this with the latest dev branch? 3d spleen seg demo is being updated to match the latest API https://github.com/Project-MONAI/tutorials/pull/246/files#diff-f5024912e8150f5b29acaf0dd10271465c1c21b0f6bc34cad41cde5703339a17

**Comment by leeh43** [comment]
Thank you for the suggestion! I am using the latest API and have changed to the latest view of coding. However, it pops up another mistake about using post_pred. 
![Screenshot from 2021-07-07 10-21-09](https://user-images.githubusercontent.com/54121206/124786443-53e4bb00-df0d-11eb-945d-a4a402076246.png)
Is there any suggestions to change it into dict format? As I followed all the steps already in the tutorial.
